apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -89,7 +88,9 @@ export default class ResearchOutputs implements ResearchOutputController {
         researchOutputCreateData.authors ?? [],
       ),
       accession: researchOutputCreateData.accession,
-      addedDate: new Date(Date.now()).toISOString(),
+      addedDate: researchOutputCreateData.published
```

Comment:
Won't this reset the added date every time the RO is saved?

Reply:
Given it's the post method, when we first create the RO it will either set the addedDate or undefined (if it's not published/ is a draft)